packages/eslint-config-prosperworks/index.js

```diff
@@ -1,22 +0,0 @@
-/* eslint-env node */
```

Comment:
Not sure what this packages folder was doing but it's not needed for an eslint-config package. By default they read index.js in the root with additional configs also existing in the root.